test/clj/rems/application/test_commands.clj

```diff
@@ -2122,3 +2128,64 @@
       (let [application (permissions/remove-role-from-user application :applicant "applicant")
             result (commands/handle-command command application injections)]
         (is (= {:errors [{:type :forbidden}]} result))))))
+
+(deftest test-redact-attachments-command
+  (let [application (apply-events nil [dummy-created-event
+                                       {:event/type :application.event/submitted
+                                        :event/time test-time
+                                        :event/actor applicant-user-id
+                                        :application/id app-id}])
+        attachment-id-1 1234
+        attachment-id-2 2345
+        injections {:valid-user? #{applicant-user-id handler-user-id "reviewer"}
+                    :find-userid identity
+                    :get-attachment-metadata
+                    {attachment-id-1 {:application/id (:application/id application)
+                                      :attachment/id attachment-id-1
+                                      :attachment/user handler-user-id}
+                     attachment-id-2 {:application/id (:application/id application)
+                                      :attachment/id attachment-id-2
+                                      :attachment/user handler-user-id}}}]
+    (testing "handler can redact attachment"
```

Comment:
We could perhaps test the case where `:redacted-attachments` is empty because it's new code and only in use here.